contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -2118,6 +2136,24 @@ get_function_name(HeapTuple tuple, TupleDesc dsc)
 	return NameGetDatum(&(func->funcname));
 }
 
+static Datum
+get_perms_schema_name(HeapTuple tuple, TupleDesc dsc)
+{
+	bool		isNull;
+	Datum		schema_name = heap_getattr(tuple, Anum_bbf_schema_perms_schema_name, dsc, &isNull);
```

Comment:
can isNull ever be true? If not, we can add an assert for the same

Reply:
No, it can't be.